tests/python/tests.py

```diff
@@ -149,8 +153,45 @@ def test1():
     print election_result(userA, election)
     print election_result(userB, election)
 
-def main():
+def test2():
+    """
+    This tests one user voting on their own election.
+    """
+    print "\n============= TEST 2 ============\n"
+    users = load_users()
+    userA = users[0]
+    userB = users[1]
+
+    election = create_election(userA, 'Triceritops Rex')
+    invite_status = invite(userA, election, userA['email'])
+    code = invite_status['code']
+    accept(userA, code)
+
+    A = create_candidate(userA, election, 'candidate-A')
+    B = create_candidate(userA, election, 'candidate-B')
+    C = create_candidate(userA, election, 'candidate-C')
+    D = create_candidate(userA, election, 'candidate-D')
+
+    votes = [
+        {'candidate_id': A['id'], 'rank': 1},
+        {'candidate_id': B['id'], 'rank': 2},
+        {'candidate_id': C['id'], 'rank': 3},
+        {'candidate_id': D['id'], 'rank': 4},
+    ]
+    bv1 = batchvote(userA, election, votes)
+    results = election_result(userA, election)
+    result_names = [result['name'] for result in results['order']]
+    assert(result_names == [u'candidate-A', u'candidate-B', u'candidate-C', u'candidate-D'])
+    print result_names
+
+def main(url = ''):
```

Comment:
@tylerharter I made tests.py accept a single optional parameter -- the url to the /api endpoint.
Example:
```
python tests.py http://pivot.app/api
```

Reply:
Also added this info to the readme.